wallet/src/main/java/com/android/identity_credential/wallet/PresentationActivity.kt

```diff
@@ -221,12 +357,14 @@ class PresentationActivity : FragmentActivity() {
                                     encodedSessionTranscript = deviceRetrievalHelper!!.sessionTranscript
                                 )
                                 deviceResponseGenerator.addDocument(documentCborBytes)
-
                             }
                             // send the response with all the Document CBOR bytes that succeeded Presentation Flows
                             sendResponseToDevice(deviceResponseGenerator.generate())
+                            _resultStatus.value = ResultStatus.SUCCESS
                         } catch (exception: Exception) {
```

Comment:
`Throwable`, not `Exception`. Also, use `e`, not `exception` as the parameter name.